### ISSUE TITLE
care_o_bot_robot dependes on EOL package ipa_canopen

### ISSUE THREAD

**Issue by cottsay** [ipa320/care-o-bot#5]
`ipa_canopen` was EOL in https://github.com/ros/rosdistro/commit/c7a3fe479b6e4f864cc685acc9671934f2732b1a and was removed from the package index, but `care_o_bot_robot` still depends on it.

Either `ipa_canopen` should be re-added, or `care_o_bot_robot` should remove the dependency and be released.

**Comment by ipa-fmw** [comment]
thanks for the hint. I'd tried to trigger a new release but failed due to not resolved keys on fedora. In care_o_bot_desktop (https://github.com/ipa320/care-o-bot/blob/indigo_dev/care_o_bot_desktop/package.xml) there are keys for ubuntu packages like language-pack-en and chromium-browser which do not exist for fedora. How do you usually deal with that? Is there a way of excluding either several keys for fedora or the whole care_o_bot_desktop package?

**Comment by cottsay** [comment]
I took a look at this, and the language pack concept doesn't apply to Fedora. All locale are present on all Fedora installs, so we can just add a key for the `filesystem` package to handle it [1].

`chromium-browser`, on the other hand, is a problem [2]. It probably won't be in Fedora any time in the near future. However, I really can't see the reason you have a dependency on this package to begin with. What is so special about chromium? Could any browser work?

Thanks,

--scott

[1] https://github.com/ros/rosdistro/pull/6815
[2] https://fedoraproject.org/wiki/Chromium

**Comment by ipa-fmw** [comment]
the care_o_bot_* packages are just metapackages without any code. We're using them to easily setup robot and desktop machines with our default development environment including a default set of packages. This is why we included chromium in the care_o_bot_desktop package.

Anyway, I found out that during the release process with bloom I could select to skip Fedora after bloom detected missing dependencies. This way all metapackages seem to build on the buildfarm: http://www.ros.org/debbuild/indigo.html?q=care
